### PR TITLE
migrated unnnic textarea component

### PR DESCRIPTION
**Pull request by cristiantela**

### CODE REVIEW

File: src/components/flow/actions/sendbroadcast/SendBroadcastForm.tsx [hunk 1]
```diff
@@ -110,13 +110,14 @@ export default class SendBroadcastForm extends React.Component<
...
           name={i18n.t('forms.message', 'Message')}
-          showLabel={false}
+          showLabel={true}
           count={Count.SMS}
           onChange={this.handleMessageUpdate}
           entry={this.state.message}
           autocomplete={true}
           focus={true}
           textarea={true}
+          maxLength={150}
```
Comment (paulobernardoaf) [inline src/components/flow/actions/sendbroadcast/SendBroadcastForm.tsx:120]: All new defined max lengths should not exist, Flows does not limit the message length like that
The design should be modified to remove that from screens

File: src/components/form/textinput/TextInputElement.tsx [hunk 2]
```diff
@@ -75,19 +80,19 @@ export default class TextInputElement extends React.Component<TextInputProps> {
...
-        // errors={this.state.errors}
-        entry={this.props.entry}
-      >
-        <div className={styles.wrapper + ' ' + styles[this.props.style || TextInputStyle.normal]}>
-          {createTextInput(this.props, this.handleChange, optional)}
-          {charCount}
-        </div>
-      </FormElement>
+      <>
+        <UnnnicTextArea
```
Comment (paulobernardoaf) [inline src/components/form/textinput/TextInputElement.tsx:84]: All text inputs will become text-areas?
What is the behavior in other card types like Call External service and Open a ticket for example?

**Review by paulobernardoaf — CHANGES_REQUESTED** [review]

File: src/components/flow/actions/sendbroadcast/SendBroadcastForm.tsx (hunk 1 above)
Comment (cristiantela) [reply src/components/flow/actions/sendbroadcast/SendBroadcastForm.tsx]: Ok, I just removed it!

**Review by paulobernardoaf — APPROVED** [review]

File: src/components/form/textinput/TextInputElement.tsx (hunk 2 above)
Comment (cristiantela) [reply src/components/form/textinput/TextInputElement.tsx:84]: This component only modifies the textarea components

**Pull request merged**